### ISSUE TITLE
FR: LDAP auth against AD group

### ISSUE THREAD

**Issue by ollyg** [netdisco/netdisco#955]
Lavin, Michael (ITS) via netdisco-users

I have a Netdisco 2.52.10 server. I have a few user accounts that use LDAP Authentication against Widows Active Directory. If there is an account for john_smith created on the Netdisco server and the LDAP Auth checkbox is checked, john_smith can log into Netdisco using the same username and password as he uses in Active Directory. That works fine.

My problem is I have a whole group of network technicians. Lets say they are all in the AD group NetworkGeniuses. They hire people and add them to the group. The geniuses get promoted and leave the group. Can I add the AD group Network_Geniuses to Netdisco so I do not have to duplicate the accounts for each person and delete the accounts for each person in Netdisco when they enter or leave the group?

In other words, can I authenticate a person by group membership in AD or do I have to mirror each individual Netdisco account in Netdisco?

**Comment by ollyg** [comment]
waiting on #1310
